GameDefinition/Content/CaveKind.hs

```diff
@@ -583,7 +591,7 @@ ambush = rogue  -- a scenario with strong missiles;
   , cwallTile     = "openableWall"
   , cstairFreq    = []
   , cstairAllowed = []
-  , cdesc         = ""
+  , cdesc         = "Scarred walls and ransacked lockers show the presence of raiders."
```

Comment:
That's good, but actually something even more sinister is going on (though I have no idea what :D ). This is the last single-level scenario. Heroes with the spaceship deed arrive, barely escaping alive, at the spaceport expecting to take possession of their ship, loot it, sell it for scrap and live happily ever after. The big Triton City is not as lawful and heavily policed as the Earth cities (though citizens are similarly completely disarmed), but that mostly means graffiti, weed, squatting, mild tax evasion and not heavily regulated casinos are a thing, but nothing else (after all, only authorities have guns, in a sealed safe and it's never been tested if the Triton Moon habitat walls can really withstand rifle bullets, as advertised, unlike any ordinary spaceship hulls). So the sewers problems and the zoo fire are absolutely unheard of disturbances and taking over the largest spaceport (think an international airport; not as sprawling, but as posh and even more hi-tech) by professional mercenaries (!!!) our of nowhere is completely off the charts.

Of course, feel free, to stand it all on its head and lead the story elsewhere.